README.md

```diff
@@ -147,6 +147,7 @@ The component accepts the following props:
 |**`customToolbar`**|function||Render a custom toolbar
 |**`customToolbarSelect`**|function||Render a custom selected rows toolbar. `function(selectedRows, displayData, setSelectedRows) => void`
 |**`customFooter`**|function||Render a custom table footer. `function(count, page, rowsPerPage, changeRowsPerPage, changePage) => string`&#124;` React Component`
+|**`customRowRender `**|function||Override default row rendering with custom function. `customRowRender({ data, dataIndex }, rowIndex) => React Component`
```

Comment:
To keep the argument structure similar to `customBodyRender`, how about using the arguments all separately, so `data, dataIndex, rowIndex`, rather than `{ data, dataIndex }, rowIndex`?

Reply:
Changed in 5744a08740562cf5dc4d0a7de75b2cbf22c0c864.